tests/data/raml_08/data-type-examples.raml

```diff
@@ -0,0 +1,77 @@
+#%RAML 0.8
```

Comment:
again, the name `data-type-*` is meant for actual Data Types. This can probably just be named `examples.raml`.